tests/integration/targets/homectl/tasks/main.yml

```diff
@@ -0,0 +1,162 @@
+# Get systemd version and if it doesn't exist don't run these tests.
```

Comment:
You need to add a file tests/integration/targets/homectl/aliases with content `unsupported` to mark that this won't run in CI.

Reply:
I think these tests can run on CI, there is a condition at bottom of test of what it should be run on:

```
  # homectl was first introduced in systemd 245 so check version  >= 245
  # As of Ubuntu 21 homed does not come with it.
  when: (systemd_version.stdout | regex_search('[0-9][0-9][0-9]') | int >= 245) and ansible_distribution != 'Ubuntu' 
```

Still needs the `unsupported` ?

Reply:
You can try to make it run in CI. Anyway, `aliases` **must exist** and have the correct entries, otherwise ansible-test ignores this target directory. How about:
```
shippable/posix/group1
skip/aix
skip/freebsd
skip/osx
skip/macos
```
That will filter out the obvious non-systemd systems.

Reply:
perfect.